cdk-for-terraform/python/Pipfile

```diff
@@ -0,0 +1,10 @@
+[[source]]
```

Comment:
We usually use `setup.py`/`setup.cfg` in combination with `virtualenv` to install the dependencies locally.

In this repo we're using Pipfile/pipenv, which is an alternative way to install dependencies - which is also perfectly fine, but breaks a bit with our usual standard pattern.

Wondering if we should unify and also adopt our standard `virtualenv` based setup here?

Reply:
I will do that in a follow-up PR 👍